sync/class.jetpack-sync-actions.php

```diff
@@ -193,11 +196,11 @@ static function do_full_sync( $modules = null ) {
 		return true;
 	}
 
-	static function minute_cron_schedule( $schedules ) {
-		if( ! isset( $schedules['1min'] ) ) {
-			$schedules['1min'] = array(
-				'interval' => 60,
-				'display' => esc_html__( 'Every minute', 'jetpack' )
+	static function jetpack_cron_schedule( $schedules ) {
+		if( ! isset( $schedules['5min'] ) ) {
+			$schedules['5min'] = array(
```

Comment:
Instead of `$schedules['5min'] `
should we set it to `$schedules[ self::DEFAULT_SYNC_CRON_INTERVAL ]`
Also would it make sense to use a call it `jetpack_sync_interval` instead of `5min` 
Since this way we can just change it to the interval and be done with it if we find that we have to update this value.